hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -254,7 +254,7 @@ coverFee_ ::
   ValidatedTx Era ->
   Either ErrCoverFee (Map TxIn TxOut, ValidatedTx Era)
 coverFee_ pparams lookupUtxo walletUtxo partialTx@ValidatedTx{body, wits} = do
-  (input, output) <- case Map.lookupMax walletUtxo of
+  (input, output) <- case Map.lookupMax (Map.filter hasEnoughValue walletUtxo) of
     Nothing ->
       Left ErrNoAvailableUtxo
```

Comment:
This error may be slightly misleading now as the wallet may have UTXO but simply not a big enough one. Shall we consider renaming it and perhaps, calculate how much is missing?